src/Test/Perf/util/runner_util.csx

```diff
@@ -38,3 +40,134 @@ async Task<ScriptState<object>> RunFile(string fileName)
     return await state.ContinueWithAsync<object>(text, scriptOptions);
 }
 
+/// Gets all csx file recursively in a given directory
+IEnumerable<string> GetAllCsxRecursive(string directoryName)
```

Comment:
We will still need skips if we ever want to benchmark csx files.  Please put skip back in.